buildSrc/src/main/kotlin/com/datadog/gradle/Dependencies.kt

```diff
@@ -16,7 +16,7 @@ object Dependencies {
         const val KronosNTP = "0.0.1-alpha09"
 
         // Android
-        const val AndroidToolsPlugin = "4.0.0"
+        const val AndroidToolsPlugin = "4.0.1"
```

Comment:
Is this necessary to embed Fresco ? Otherwise, maybe we should upgrade this in a dedicated PR ?

Reply:
Yup...I will drop that commit, we don't need it actually